ui/k-info-panel/show_circuit.kytos

```diff
@@ -330,11 +331,15 @@ module.exports = {
          {'value': uni_a_data['port_name']}, 
          {'value': uni_z_data['port_name']}]
       };
+
+      // Tag is optional, but can be edited
+      let tag_a = {'value': "", 'editable':true, 'name':'tag_a'};
+      let tag_b = {'value': "", 'editable':true, 'name':'tag_b'};
       if(this.circuit['uni_a']['tag']) {
-        let tag_a = {'value': this.circuit['uni_a']['tag']['value'], 'editable':true, 'name':'tag_a'};
-        let tag_b = {'value': this.circuit['uni_z']['tag']['value'], 'editable':true, 'name':'tag_b'};
-        this.endpoints_data['Tag Value'] = [tag_a, tag_b];
+        tag_a['value'] = this.circuit['uni_a']['tag']['value'];
```

Comment:
@rmotitsuki, I think we'll have to check individually if both `['uni_a']['tag']` and `['uni_z']['tag']` are truthy, otherwise we'd have a problem where for example only the `uni_z` has a tag, this is a valid EVC payload for instance:

```
{
    "name": "evc1",
    "dynamic_backup_path": true,
    "uni_a": {
        "interface_id": "00:00:00:00:00:00:00:01:1"
    },
    "uni_z": {
        "interface_id": "00:00:00:00:00:00:00:03:1",
        "tag": {
            "tag_type": 1,
            "value": 1000
        }
    }
}
```

Could you apply this patch or double check another way to ensure this?

```
diff --git a/ui/k-info-panel/show_circuit.kytos b/ui/k-info-panel/show_circuit.kytos
index 1b4503f..1c43afd 100644
--- a/ui/k-info-panel/show_circuit.kytos
+++ b/ui/k-info-panel/show_circuit.kytos
@@ -337,6 +337,8 @@ module.exports = {
       let tag_b = {'value': "", 'editable':true, 'name':'tag_b'};
       if(this.circuit['uni_a']['tag']) {
         tag_a['value'] = this.circuit['uni_a']['tag']['value'];
+      }
+      if(this.circuit['uni_z']['tag']) {
         tag_b['value'] = this.circuit['uni_z']['tag']['value'];
       }
       this.endpoints_data['Tag Value'] = [tag_a, tag_b];
```

The original issue is gone btw, awesome, thank you.

Reply:
Ok. I will fix it.